pkg/ingress/kube/ingress/controller.go

```diff
@@ -568,11 +581,13 @@ func (c *controller) ConvertHTTPRoute(convertOptions *common.ConvertOptions, wra
 
 			// host and path overlay check across different ingresses.
 			hostAndPath := wrapperHttpRoute.BasePathFormat()
-			if preIngress, exist := convertOptions.HostAndPath2Ingress[hostAndPath]; exist {
+			arr = append(arr, hostAndPath)
+			key := strings.Join(arr, ",")
+			if preIngress, exist := convertOptions.HostAndPath2Ingress[key]; exist {
```

Comment:
This map is no longer “HostAndPath2Ingress”.

Reply:
ok, I've reacotor it.